app/controllers/api/v1/moves_controller.rb

```diff
@@ -3,46 +3,44 @@
 module Api
   module V1
     class MovesController < ApiController
+      before_action :validate_filter_params, only: %i[index]
+
       def index
-        moves_params = Moves::ParamsValidator.new(filter_params, params[:sort] || {})
-        if moves_params.valid?
-          moves = Moves::Finder.new(filter_params, current_ability, params[:sort] || {}).call
-          # Excludes potentially many court hearing documents to reduce the request size. This was requested specifically by the frontend team.
-          paginate moves, include: MoveSerializer::INCLUDED_ATTRIBUTES.dup.except(:court_hearings), fields: MoveSerializer::INCLUDED_FIELDS
-        else
-          render json: { error: moves_params.errors }, status: :bad_request
```

Comment:
with this change we are moving away from `:bad_request` 400 errors on validation and instead returning a 422, since we seem to be following this pattern in other controllers I was wondering why it was different here?

Reply:
Although I guess 422 is probably the correct error to be displaying for validations

Reply:
so yes, starting with the journeys controller we're moving towards validation errors being 422 rather than 400 - cc @alexdesi 
we can look at other controllers as time allows